lib/openshift/cluster_logging.rb

```diff
@@ -53,7 +76,7 @@ class ClusterLogging < ClusterResource
     end
 
     private def es_cluster_health(user: nil, cached: true, quiet: false)
-      es_status_raw(user: user, cached: cached, quiet: quiet).first['clusterHealth']
+      es_status_raw(user: user, cached: cached, quiet: quiet).first['cluster']['status']
```

Comment:
@QiaolingTang this change seem to work for 4.2 but for version <= 4.1?

Reply:
@pruan-rht  It works for 4.2, and I don't think it will work for 4.1.

Reply:
Ok, instead of hardcoding to `['cluster']['status']`, you can check whether `['clusterHealth']` exist and use `['cluster']['status']` when needed.

Reply:
Thanks @akostadinov , I updated this file per your advice, PTAL.